### PR TITLE
OBSDOCS-205 - CLF audit log filtering

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.12+

Issue: [OBSDOCS-205](https://issues.redhat.com//browse/OBSDOCS-205)

Link to docs preview: https://70608--ocpdocs-pr.netlify.app/openshift-enterprise/latest/logging/log_collection_forwarding/configuring-log-forwarding#logging_audit_filtering_configuring-log-forwarding

QE review: @anpingli / @kabirbhartiRH / @QiaolingTang 

- [x] QE has approved this change.

Additional information:
SME review: @alanconway / @jcantrill

### CODE REVIEW

File: modules/logging-audit-log-filtering.adoc
Comment (QiaolingTang) [inline modules/logging-audit-log-filtering.adoc:16]: Seems `oc list` is not a valid command, I got:

```
$ oc list
error: unknown command "list" for "oc"

Did you mean this?
	get
	wait
```

**Comment by snarayan-redhat** [comment]
@libander I would suggest submitting this for peer review once you get a QE approval.

**Comment by JoeAldinger** [comment]
@libander do you mind adding the assembly file so the preview works, please?

File: modules/logging-audit-log-filtering.adoc
Comment (libander) [reply modules/logging-audit-log-filtering.adoc]: @QiaolingTang - fixed, thank you!

**Comment by QiaolingTang** [comment]
LGTM.

**Review by snarayan-redhat — COMMENTED** [review]
Good effort. I have left few suggestions which might want to consider.

**Comment by snarayan-redhat** [comment]
For now, I am not adding a `peer-review-done` label since there are some comments that need to be addressed. You can request for a review once ready by adding `peer-review-needed` label again.

**Review by jcantrill — COMMENTED** [review]
/lgtm

File: modules/logging-audit-log-filtering.adoc
Comment (libander) [reply modules/logging-audit-log-filtering.adoc:3]: Good catch, thanks!

**Review by dfitzmau — COMMENTED** [review]
Hi @libander . Nice work on the contnet. I added some minor comments but the YAML file might indicate a bigger issue for customers that copy then example verbatim. I see some red text highlighting in the preview that might indicate an issue.

**Comment by libander** [comment]
> Hi @libander . Nice work on the contnet. I added some minor comments but the YAML file might indicate a bigger issue for customers that copy then example verbatim. I see some red text highlighting in the preview that might indicate an issue.

Thanks! Added admonitions to those fields, and corrected the admonition. That example is not intended to be copied as it isn't a recommended policy. :)

File: modules/logging-audit-log-filtering.adoc
Comment (mburke5678) [inline modules/logging-audit-log-filtering.adoc:31]:
```suggestion
The example provided is intended to illustrate the range of rules possible in an audit policy and is not a recommended configuration.
```

File: modules/logging-audit-log-filtering.adoc
Comment (mburke5678) [inline modules/logging-audit-log-filtering.adoc:47]: Would it make sense to add some examples to the `inputRefs` and `filterRefs` parameters, especially where the code example is purely illustrative. It took me several minutes to figure it out (assuming I did):

```
  pipelines:
    - name: my-pipeline
      inputRefs: audit #<1>
      filterRefs: my-policy #<2>
```

**Comment by mburke5678** [comment]
@libander I have a couple of nits I was hoping you could look at before we merge.

**Review by mburke5678 — APPROVED** [review]

**Pull request merged**
